index.html

```diff
@@ -360,10 +360,12 @@ <h3>Ed25519VerificationKey2020</h3>
             }
           </pre>
 
-          <pre class="example" title="Example in DID Document.">
+          <pre class="example" title="Ed25519VerificationKey2020 in a DID Document.">
             {
               "@context": [
                 "https://www.w3.org/ns/did/v1",
+                "https://w3id.org/security/v1",
```

Comment:
```suggestion
```

Reply:
I don't know if it's a problem with the [JSON-LD playground](https://json-ld.org/playground/), but without "https://w3id.org/security/v1" in context I only get the four statements that I get without my PR.

Reply:
Yes, you need to remove "https://w3id.org/security/v1"... you don't need it here.

Reply:
But @msporny, without `https://w3id.org/security/v1`, as per [my comment below](https://github.com/w3c-ccg/di-eddsa-2020/pull/15#issuecomment-1325680415), all the following statements about the public key are missing:

```ttl
<did:example:123> <https://w3id.org/security#publicKey> <did:example:123#key-0> .
<did:example:123#key-0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/security#Ed25519VerificationKey2020> .
<did:example:123#key-0> <https://w3id.org/security#controller> <did:example:123> .
<did:example:123#key-0> <https://w3id.org/security#publicKeyMultibase> "z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP"^^<https://w3id.org/security#multibase> .
```

Therefore what do you mean by "the `https://w3id.org/security/v1` context is not needed?

Reply:
The problem with this example is that it uses `publicKey` when it should instead use `verificationMethod`. That should remove the need for the `security/v1` context and make it conformant with the DID spec (`publicKey` was used in very early versions of the DID spec and it was later changed to `verificationMethod`).